packages/@aws-cdk-testing/cli-integ/lib/package-sources/find-root.ts

```diff
@@ -0,0 +1,14 @@
+import * as path from 'path';
+import { findUp } from '../files';
+
+/**
+ * Find the root directory of the repo from the current directory
+ */
+export async function autoFindRoot() {
```

Comment:
Maybe this? Because it could also be package root.

```suggestion
export async function autoFindRepoRoot() {
```